generatebundlefile/hack/generate_bundle.sh

```diff
@@ -46,10 +46,8 @@ function generate () {
 function push () {
     local version=$1
     cd "${BASE_DIRECTORY}/generatebundlefile/output"
-    awsAuth "ecr-public" | "$ORAS_BIN" push --password-stdin \
-        "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
-    awsAuth "ecr-public" | "$ORAS_BIN" push --password-stdin \
-        "${REPO}:v${version}-latest" bundle.yaml
+    awsAuth "ecr-public" | "$ORAS_BIN" push "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
+    awsAuth "ecr-public" | "$ORAS_BIN" push "${REPO}:v${version}-latest" bundle.yaml
```

Comment:
```suggestion
    awsAuth "ecr-public"
    "$ORAS_BIN" push "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
    "$ORAS_BIN" push "${REPO}:v${version}-latest" bundle.yaml
```

Reply:
That is how it used to be. I was just wondering why you were piping the output of that to `oras push`. I think what you have will work fine, it is just a bit odd.

Reply:
I think it's because of the changes Eric made to the `common.sh` it required being piped now.